### PR TITLE
Feature/sr support

### PR DESCRIPTION
**Pull request by siphiuel**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -49,9 +49,6 @@
 [submodule "vendor/nim-eth"]
 	path = vendor/nim-eth
 	url = https://github.com/status-im/nim-eth.git
-[submodule "vendor/nim-secp256k1"]
```
Comment (richard-ramos) [inline .gitmodules:52]: Why was nim-secp256k1 removed?

Comment (siphiuel) [reply .gitmodules:52]: Because it's duplicated.

**Comment by michaelsbradleyjr** [comment]
For the record, (1) I'm not sure how I closed this PR and (2) however it happened it was not done purposefully. It should be re-opened or a new one can be created to take its place. Sorry, wish I knew what happened!

**Comment by siphiuel** [comment]
Re-created in #107

**Pull request closed without merging**